src/BugsnagUnity/Native/Android/NativeEvent.cs

```diff
@@ -148,5 +148,6 @@ public void SetUser(string id, string email, string name)
         {
             NativePointer.Call("setUser", id, email, name);
         }
+
```

Comment:
Seems like an unnecessary change 😂

Reply:
I was going to do callbacks in the same PR but changed my mind and removed some code that was previously checked in